tests/xdc/advanced_visibility_test.go

```diff
@@ -153,6 +153,7 @@ func (s *AdvVisCrossDCTestSuite) SetupSuite() {
 	s.Require().NoError(err)
 	// Wait for cluster metadata to refresh new added clusters
 	time.Sleep(time.Millisecond * 200)
+	time.Sleep(time.Second * 10)
```

Comment:
Is this `tests.NamespaceCacheRefreshInterval`, or is it different? If it is, then maybe express the sleep time as a function of that?

Also, looks like we can get rid of the 200ms sleep now.

Reply:
Oh sorry, didn't see this was in Draft status!